doc-tool/src/dotty/tools/dottydoc/staticsite/MarkdownLinkVisitor.scala

```diff
@@ -7,14 +7,21 @@ import com.vladsch.flexmark.util.sequence.CharSubSequence
 import model.{ Package, NonEntity, Val, Def, TypeAlias }
 import dottydoc.util.MemberLookup
 
+import dotty.tools.dotc.core.Contexts.Context
+import dotty.tools.dotc.core.Decorators._
+
 object MarkdownLinkVisitor {
   private val EntityLink = """([^\.]+)(\.[^\.]+)*""".r
-  def apply(node: Node, docs: Map[String, Package], params: Map[String, AnyRef]): Unit =
+  def apply(node: Node, docs: Map[String, Package], params: Map[String, AnyRef])(implicit ctx: Context): Unit =
     (new NodeVisitor(
       new VisitHandler(classOf[Link], new Visitor[Link] with MemberLookup {
         override def visit(node: Link): Unit = {
           val url = node.getUrl
-          if (url.endsWith(".md")) node.setUrl {
+          val x = new java.net.URI(url.toString)
+          val path = x.getPath()
+          val method = path.substring(path.lastIndexOf('/')+1).dropRight(3);
+          val method1 = method.toTypeName
+          if ((url.endsWith(".md")) && (ctx.getClassIfDefined(method1).exists == false)) node.setUrl{
```

Comment:
Ah sorry, that was more out of despair that by some unearthly force, it would work. Sadly it didn't :/